src/crewai/experimental/evaluation/agent_evaluator.py

```diff
@@ -1,17 +1,18 @@
+import threading
```

Comment:
couple type errors here:
<img width="762" height="383" alt="Screenshot 2025-07-15 at 12 12 08 PM" src="https://github.com/user-attachments/assets/1422fff1-f109-4ea7-8e4b-0651c0164430" />

Reply:
type-checking is on basic for me

Reply:
Same here! 🙂 I'm just wondering if we should rely on your local type-checking setup or the one used in CI.. just to keep things consistent. Either option works for me!

Maybe we missing some setup on mypy to make it better